src/CitySwitcherModal.css

```
@@ -148,353 +111,308 @@ body.show-city-picker .city-switcher-modal__geocoderMount {
+.theme-light .city-switcher-modal__placeSearchIcon--matched {
+  color: rgba(0, 0, 0, 0.62);
 }
 
-.city-switcher-modal__citiesGrid {
-  display: grid;
-  grid-template-columns: repeat(2, minmax(0, 1fr));
-  gap: 10px;
-  /* padding: 0 16px 12px; */
+.city-switcher-modal__placeSearchPrimary {
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)